tuna/fin_eval.py

```diff
@@ -76,10 +77,13 @@ def fin_pdb_input(self, _fjob):
     with DbSession() as session:
       perf_compile_res = []
 
+      # pylint: disable=comparison-with-callable
```

Comment:
hmmm interesting this seems to be an issue with pylint not recognizing the declared_attr decorator we have for session or solver. Hopefully they fix/add something to avoid this in the future, until then it seems like we have to keep these disable hints